### DISCUSSION TITLE
Local Media Player

### DISCUSSION THREAD

**Discussion by MSOB7YY** [ReVanced/revanced-discussions#148] [Question] [answered]
app can play local media files as videos and music in a youtube ui

**Comment by ArjixWasTaken** [comment, marked as answer]
youtube music already allows you to play local files
so this request can focus on just the video part

**Comment by TheJeterLP** [comment]
To be honest, just use the gallery for that.

**Reply by ArjixWasTaken** [reply]
I'd suggest VLC instead, but yeah

**Reply by MSOB7YY** [reply]
nah, something about youtube ui makes it better (home page, recommendations, playlists) i wish it could manage local videos too
